src/index.ts

```diff
@@ -100,36 +99,36 @@ export function httpTriggerWrapper (fn, customDimensions = {}) {
  * @see https://github.com/microsoft/ApplicationInsights-node.js/#setting-up-auto-correlation-for-azure-functions
  */
 export function functionWrapper(fn, eventName = "FUNCTION_EXECUTION",  customDimensions = {}) {
-    return async function contextPropagationTrigger(context) {
-        const correlationContext = ai.startOperation(context, context.executionContext.functionName)
-        return ai.wrapWithCorrelationContext(async () => {
-            const startTime = Date.now()
-            try {
-                await fn(context)
-                ai.defaultClient.trackRequest({
-                    name: context.executionContext.functionName,
-                    resultCode: 'complete',
-                    success: true,
-                    url: eventName,
-                    duration: Date.now() - startTime,
-                    id: correlationContext.operation.parentId,
-                    properties: customDimensions
-                })
```

Comment:
What’s going on with the spacing changes? Do you have eslint setup?

Reply:
@zhughes3 I'm going to address the formatting in a seperate PR.

Reply:
Do I really have to revert these changes? i just brought the indenting changes in line with the rest of the file manually, instead of what was copied and pasted from Azure.

Reply:
yes. We need a clean PR. I am merging in #14